compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -1268,19 +1261,16 @@ OMR::ResolvedMethodSymbol::genIL(TR_FrontEnd * fe, TR::Compilation * comp, TR::S
             if (doOSR)
                {
                TR_ASSERT(comp->getOSRCompilationData(), "OSR compilation data is NULL\n");
-               if (comp->canAffordOSRControlFlow())
+               self()->genAndAttachOSRCodeBlocks(comp->getCurrentInlinedSiteIndex());
```

Comment:
Could you provide the logic behind removing the canAffordOSRControlFlow query here - we won't end up generating more code after this change than we did before right?